kafka_utils/kafka_rolling_restart/main.py

```diff
@@ -367,6 +412,34 @@ def validate_opts(opts, brokers_num):
     return False
 
 
+def get_task_class(tasks, task_args):
+    """Reads in a list of tasks provided by the user,
+    loads the appropiate task, and returns two lists,
+    pre_stop_tasks and post_stop_tasks
+    :param tasks: list of strings locating tasks to load
+    :type tasks: list
+    :param task_args: list of strings to be used as args
+    :type task_args: list
+    """
+    pre_stop_task = []
+    post_stop_task = []
+    task_args_itr = task_args.__iter__()
+    tasks_classes = [PreStopTask, PostStopTask]
+
+    for func in tasks:
```

Comment:
I think that if you use enumerate here, it gets rid of you having to use task_args_iter, alternatively you could do:

task_to_task_args = dict(zip(tasks, task_args))
for func, task_args in task_to_task_args.items():
    ....